policyengine_us/parameters/gov/states/la/tax/income/exemptions/amount/blind_or_disabled.yaml

```diff
@@ -0,0 +1,15 @@
+description: Louisiana provides an exemption of this amount for disabled and blind filers.
+
+values:
+  2014-01-01: 1_000
+metadata:
+  reference:
+    - title: Louisiana Laws Revised Statutes, Title 47 — Revenue and taxation, RS 47:79 — Credits of individuals against net income §79, A, (2)
+      href: https://www.legis.la.gov/legis/Law.aspx?d=102518
+    - title: "Form IT-540-WEB "
```

Comment:
What page and line number? I don't see $1,000 here.

Reply:
the file doesn't contain the value $1000, the first one in reference has $1000 in it. Should I delete the other references?

Reply:
yes delete the reference that does not have it